.github/workflows/php.yml

```diff
@@ -3,27 +3,12 @@ on: [push, pull_request]
 jobs:
   # Check there is no syntax errors in the project
   php-linter:
-    name: PHP Syntax check 5.6 => 8.1
+    name: PHP Syntax check 8.1
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2.0.0
 
-      - name: PHP syntax checker 5.6
```

Comment:
@patrocle I think it would be better to keep PHP checks for 7.2 to 8.1. As you probably know, PrestaShop 8.0 is compatible with 7.2 to 8.1.